R/geom-.r

```diff
@@ -108,12 +108,24 @@ Geom <- ggproto("Geom",
 
   setup_data = function(data, params) data,
 
+  # evaluate defaults according to theme
+  eval_defaults = function(self, theme) {
+
+    if (length(theme) == 0) theme <- theme_grey()
```

Comment:
Should this retrieve the currently set default theme instead of `theme_grey()`?

Reply:
yes. good catch. will fix.